src/ray/common/runtime_env_manager.h

```diff
@@ -19,6 +19,11 @@
 
 namespace ray {
 
+// Return whether a string representation of a runtime env represents an empty
+// runtime env.  It could either be "" (from the default string value in protobuf),
+// or "{}" (from serializing an empty Python dict or a JSON file.)
+bool IsRuntimeEnvEmpty(const std::string &serialized_runtime_env);
```

Comment:
Can you move it to a common, not runtime_env_manager.h? I think runtime_env_manager.h should be higher layer than things like TaskSpec

Reply:
agree, good point

Reply:
@rkooo567 @edoakes Updated, how does it look now?

Reply:
LGTM, not sure if sang had a specific place in mind

Reply:
Yeah no preference, tho we can probably name the file just common.h (since it is already under runtime env folder)